src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+const ADD_BOOK = 'bookstore-react-app/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore-react-app/books/REMOVE_BOOK';
+
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          id: action.id,
+          title: action.title,
+          author: action.author,
+        },
+      ];
+
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.id);
+
+    default:
+      return state;
+  }
+};
+
+export const addBook = (bookTitle, bookAuthor, bookId) => {
+  const book = {
+    type: ADD_BOOK,
+    title: bookTitle,
+    author: bookAuthor,
+    id: bookId,
+  };
+  return book;
+};
```

Comment:
- [ ] **[OPTIONAL]** Since a book would be made of an id, title, and author, is there really any need to define all its attributes this way? We can have a more concise definition of the book item, like so:

  ```
  export const addBook = (book) => {
    const book = {
      type: ADD_BOOK,
      payload: book
    };
    return book;
  };
  ```